src/org/etools/j1939_84/controllers/part04/Part04Step05Controller.java

```
@@ -53,10 +62,34 @@
     @Override
     protected void run() throws Throwable {
         // 6.4.5.1.a DS DM23 [(send Request (PGN 59904) for PGN 64949 (SPNs 1213-1215, 1706, and 3038)]) to each OBD ECU.
-        // 6.4.5.2.a Fail if any ECU reports > 0 previously active DTC.
-        // 6.4.5.2.b Fail if any ECU reports a different MIL status than it did in DM12 response earlier in this part.
```

6.4.5.2.b failure is not reported.